mm2src/mm2_main/tests/docker_tests/docker_tests_inner.rs

```diff
@@ -5517,3 +5517,84 @@ fn test_orderbook_depth() {
     block_on(mm_bob.stop()).unwrap();
     block_on(mm_alice.stop()).unwrap();
 }
+
+#[test]
+fn test_approve_erc20() {
```

Comment:
Maybe for this test we should call approve first then check that allowance is the right amount after approve call.

Reply:
changed